src/main/scala/scorex/core/network/NodeViewSynchronizer.scala

```diff
@@ -31,16 +31,16 @@ import scala.util.Success
   * @tparam SIS SyncInfoMessage specification
   */
 class NodeViewSynchronizer[TX <: Transaction,
-                           SI <: SyncInfo,
```

Comment:
is it automatic code formatting who aligns these lines that way?

Reply:
yes